.github/workflows/test.yml

```
@@ -37,7 +37,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [
-          "macos-14",
+          "macos-14-arm64",
```

No runners are found for `macos-14-arm64`, and `macos-14` is already M1/ARM64 (and only M1):

https://github.blog/changelog/2024-01-30-github-actions-macos-14-sonoma-is-now-available/

```suggestion
          "macos-14",
```

---

Thanks. The problem is there is no a list of available runners in documentation, and [in the repository](https://github.com/actions/runner-images/tree/main/images/macos)  `macos-14-arm64` is present.